### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by lulco**
Test failed on my local machine, so I just want to run them on GA

### CODE REVIEW

**Comment by lulco** [comment]
Ok, they failed here too. So something happend somewhere. Any idea? May be it is connected with https://github.com/revealphp/reveal-src/issues/22 ?

File: packages/template-phpstan-compiler/composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "nette/utils": "^3.2",
         "symplify/astral": "^10.0",
         "symplify/package-builder": "^10.2",
-        "phpstan/phpstan": "^1.5.6"
+        "phpstan/phpstan": ">=1.5.6 <=1.7.15"
```
Comment (TomasVotruba) [inline packages/template-phpstan-compiler/composer.json:11]: There should be bumped to PHPStan 1.8 as well

File: composer.json [hunk 2]
```diff
@@ -7,10 +7,10 @@
         "php": ">=8.1",
         "latte/latte": "^2.11",
         "nette/utils": "^3.2",
-        "phpstan/phpstan": "^1.7.10",
+        "phpstan/phpstan": "~1.7.10",
         "symfony/console": "^6.1",
         "symfony/dependency-injection": "^6.1",
-        "symplify/astral": "^10.2",
+        "symplify/astral": "~10.2.0",
```
[the remark this answers is not in the document]
Comment (lulco) [reply composer.json:13]: Yes, I know, but while there is no solution for https://github.com/revealphp/reveal/issues/1 this package is unstable

File: packages/template-phpstan-compiler/composer.json (hunk 1 above)
Comment (lulco) [reply packages/template-phpstan-compiler/composer.json:11]: I wanted to make this package working again and then update astral and phpstan to newest versions.

With phpstan 1.8 I got error:
```
PHPStan\Analyser\FileAnalyser::analyseFile(): Argument #4 ($collectorRegistry) must be of type PHPStan\Collectors\Registry, null given, called in LatteCompleteCheckRule.php
```

Comment (TomasVotruba) [reply packages/template-phpstan-compiler/composer.json:11]: That must be upgraded then. We'll use the latest PHPStan 1.8 along with Symplify and Rector.

Comment (lulco) [reply packages/template-phpstan-compiler/composer.json]: ok phpstan is bumped to 1.8, but what should I do with missing Symplify\Astral\NodeFinder\SimpleNodeFinder? Are there some replacements?

**Comment by TomasVotruba** [comment]
Mergin as the CI is passing :+1:

File: packages/template-phpstan-compiler/composer.json (hunk 1 above)
Comment (TomasVotruba) [reply packages/template-phpstan-compiler/composer.json]: See the PHPStan release info: https://phpstan.org/blog/preprocessing-ast-for-custom-rules

For some cases, custom node visitor is needed.
In other case, the rule can be rewritten not to need the parent/next nodes.

**Comment by TomasVotruba** [comment]
Thank you for the fixed :clap:

**Pull request merged**